### ISSUE TITLE
Version 1.5.0 quits right after launch

### ISSUE THREAD

**Issue by timoisalive** [mesmotronic/air-ane-fullscreen#34]
The app quits as soon as it's launched (shows black screen for half a second)

I'm only including the 1.5.0 ANE, not even referring to it in my project.

I've tried a release and debug build.

Developed with:
AIR 24.0.0.180
IntelliJ IDEA

Tried with:
Nexus 5 6.0.1
OnePlus One 6.0.1

Could the Android minimum/target SDK version be a problem? My current setting is:

`<uses-sdk android:minSdkVersion="9" android:targetSdkVersion="21" />`

**Comment by neilrackett** [comment]
We've just tried installing the example Flex app using the 1.5.0 ANE on devices running 5.1.1, 6.0.1 and 7.1.1 and it worked without any problems.

Can you double check that the ANE is being packaged into your app correctly? The only way we were able to create a similar issue was by including the extension in the `app.xml`, but excluding it from the build packaging options.

**Comment by timoisalive** [comment]
Ok, I believe the build packaging options had to be correct, because just by replacing the ANE file back to the old one everything worked fine. But let me try again next week, have a nice weekend! :)

**Comment by neilrackett** [comment]
I've updated and rebuilt the[1.5.0 release](https://github.com/mesmotronic/air-ane-fullscreen/releases/edit/v1.5.0) against the latest Android SDK (API 25) using the latest Android build tools - can you see if that fixes the issue on your devices?

**Comment by timoisalive** [comment]
Thanks for looking into it @neilrackett!

Unfortunately still the same problem. I'm only replacing the old ane (works ok) with this new one and the app quits right at launch... :(

I'll make a simple test project to see if there's something in my project that's causing the crash.

**Comment by timoisalive** [comment]
Ok, I'm sorry I didn't see this before. When I debug in the emulator, I get this message:

`[AIR Debug Launcher]: Error: The extension com.mesmotronic.ane.fullscreen has either a namespace version or library.swf with a version that is incompatible with the applications namespace or root SWF.
`

This is with 1.5.0, not the old one.

**Comment by timoisalive** [comment]
Is the latest release by the way strictly AIR 24 only? We have to ship with AIR 23, so I'm going to have to wait to use this...

**Comment by neilrackett** [comment]
Can you confirm that you're using AIR SDK 24 and not setting any SWF or player versions in your build command?

**Comment by timoisalive** [comment]
I am testing with AIR 24 (build 180). No version settings in the build command.

Here's my app descriptor:

`
<?xml version="1.0" encoding="utf-8" ?>

<application xmlns="http://ns.adobe.com/air/application/24.0">

    <id>com.beigeelephant.thisissand</id>
    <filename>Main</filename>
    <name>Main</name>
    <versionNumber>2.3.0</versionNumber>

    <initialWindow>
        <content></content>
        <visible>true</visible>
        <autoOrients>true</autoOrients>
        <fullScreen>false</fullScreen>
        <renderMode>direct</renderMode>
        <depthAndStencil>true</depthAndStencil>
        <softKeyboardBehavior>none</softKeyboardBehavior>
    </initialWindow>

    <android>
        <manifestAdditions>
            <![CDATA[
			<manifest android:installLocation="auto">
				<uses-sdk android:minSdkVersion="14" />
				<application android:hardwareAccelerated="false">
				</application>
			</manifest>
			]]>
        </manifestAdditions>
    </android>
    
    <extensions>
        <extensionID>com.mesmotronic.ane.fullscreen</extensionID>
    </extensions>

</application>
`

**Comment by neilrackett** [comment]
Hmmmm... Looks fine to me.

Look forward to finding out if you have the same result with a simple test project.

**Comment by timoisalive** [comment]
This was the simple test project... :(

**Comment by lukejaylarsen** [comment]
I have the same issue as @neilrackett ... tried from air 24 to 26, but no luck

**Comment by neilrackett** [comment]
@lukejaylarsen Are you able to share the DDMS output from your app? That should tell us what's causing the crash.

From the command line: Type `ddms` (Windows) or `./ddms` (Mac/Linux) from the Android SDK `tools` folder.

**Comment by llorenzo** [comment]
Hello, any news to that? I cant compile for SDK 28.
